ct-app/postman/postman_tasks.py

```diff
@@ -81,12 +90,34 @@ async def async_send_1_hop_message(
     )
 
     # node is reachable, messages can be sent
-    for _ in range(count):
-        state = await api.send_message(address, "foo", [peer_id])  # noqa: F841
+    for idx in range(count):
+        state = await api.send_message(address, "foo", [peer_id])
+
+        if not state:
+            log.error("Could not send message")
+            break
+    else:
+        log.info(
+            f"{count} messages sent to `{peer_id}` "
+            + f"via {node_list[node_index]}(#{node_index} - {api_host})"
+        )
 
-    log.info(
-        f"{count} messages sent to `{peer_id}` "
-        + f"via {node_list[node_index]}(#{node_index} - {api_host})"
+        return "SUCCESS"
+
+    log.error("Could not send all messages")
```

Comment:
This one doesn't seem to be fatal ... maybe `warn`?

Reply:
Agreed. Resolved in new commit.